### PR TITLE
Add travis.ci support and custom script.

### PR DESCRIPTION
**Pull request by CosmicWebServices**

### CODE REVIEW

**Comment by themightychris** [comment]
Looks great! Let's have it also loop through all our yaml files and run it through the sorter, then report an error if that dirties the git tree so we can notify the commiter that their files need to be presorted to facilitate review

**Comment by CosmicWebServices** [comment]
@themightychris Ok I will do that not very sure on how to report errors.

File: script.sh [hunk 1]
```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
+FILES=*.yml
 gem install yaml-lint
-yaml-lint topics.yml #Recomended: Put all the yml files in a folder such as yml so you can do yaml-lint yml so you do not have to go through all the trouble of adding each file one by one.
-sort_yaml < topics.yml > topics.sorted.yml && mv topics.sorted.yml topics.yml
+for f in $FILES
+do
+  echo "Processing $f"
+  yaml-lint $f
+  sort_yaml < $f > $f && mv $f $f
```
Comment (themightychris) [inline script.sh]: @CosmicWebServices this isn't going to work, the temporary file needs a prefix/suffix

**Comment by themightychris** [comment]
Looking good, now to have Travis CI see the build as failed if the repo becomes dirty from this. I imagine that will involve writing messages to STDERR and exiting with a nonzero status

**Comment by CosmicWebServices** [comment]
Your requested changes from a while ago are completed (well they were for a little while).

**Comment by themightychris** [comment]
This looks like it would be cool to incorporate, dumping it here: https://github.com/bluzi/travis-buddy

Also following some conversations with Code for SF's project matchmaking team this week, we want to implement some additional checks on the changes a PR makes to the data like ensuring that all identifiers either still exist or have been moved to the synonyms list of another entry

**Comment by themightychris** [comment]
Closing this issue because we're changing the format, and GH Actions would be the best place to do this now

**Pull request closed without merging**